cmd/podman/kube/generate.go

```diff
@@ -68,10 +70,32 @@ func generateFlags(cmd *cobra.Command) {
 	flags.StringVarP(&generateFile, filenameFlagName, "f", "", "Write output to the specified path")
 	_ = cmd.RegisterFlagCompletionFunc(filenameFlagName, completion.AutocompleteDefault)
 
+	typeFlagName := "type"
+	flags.StringVarP(&generateOptions.Type, typeFlagName, "t", define.PodKind, "Generate YAML for the given Kubernetes kind")
+	_ = cmd.RegisterFlagCompletionFunc(typeFlagName, completion.AutocompleteNone)
+
+	replicasFlagName := "replicas"
+	flags.Int32VarP(&generateOptions.Replicas, replicasFlagName, "r", 0, "Set the replicas number for Deployment kind")
```

Comment:
Should the default replicas be 1? I'm not a Kubernetes expert but think I read that having a replica of 0 means that no pods will run in the deployment which doesn't seem right.

Reply:
If the user explicitly sets replicas to 0, would the replicas correctly populate as 0?

Reply:
No it won't, 0 is a wrong value and replicas have to be 1 or more. I am assuming users are smart, but I guess I can throw an error if someone tries to set replicas to less than 1.

Reply:
I think 0 is valid, as it's listed in the kube deployment spec? But I haven't tried this out myself. https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#deploymentspec-v1-apps

Reply:
Fixed it to default to 1 and throw an error if a user tries to set it to less than 1. Also added a condition to only add the `replicas: x` line in the generated yaml only if the replicas value is greater than 1.